qa-tests-backend/src/test/groovy/VulnMgmtTest.groovy

```diff
@@ -256,7 +256,14 @@ fragment cveFields on ImageVulnerability {
 
         def embeddedImageRes = gqlService.Call(getEmbeddedImageQuery(),
                 [id: imageDigest, query: "CVE:CVE-2017-10684"])
-        assert embeddedImageRes.hasNoErrors()
+
+        // Expanded instead of using hasErrors() for easier debugging if there are errors
+        // as the test framework will actually print out the errors now
+        assert embeddedImageRes.code == 200
```

Comment:
Is the concern only with the above graphQL call?

Reply:
This is the one that has failed. Others might, but I'm hoping given this one has for sure we can get logs out of it.